server/subject_transform.go

```diff
@@ -355,7 +364,7 @@ func (tr *subjectTransform) TransformSubject(subject string) string {
 
 func (tr *subjectTransform) getHashPartition(key []byte, numBuckets int) string {
 	h := fnv.New32a()
-	h.Write(key)
+	_, _ = h.Write(key)
```

Comment:
What is this?

Reply:
Avoids a warning (it annoyed me because it was the only non-weak warning in the file :))